server/src/main/java/io/spine/examples/chatspn/server/chat/ChatAggregate.java

```diff
@@ -107,7 +110,8 @@ private void event(GroupChatCreated e) {
      *
      * @return {@link MembersRemoved} if at least one member was removed
      * @throws MembersCannotBeRemoved
-     *         if chat isn't a group,
+     *         if chat deleted,
```

Comment:
"if chat (what?) deleted"? "...is deleted"?

Right now it is "chat" who deleted something or someone.